Sprint-1/exercises/initials.js

```diff
@@ -4,3 +4,6 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string.
 // This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
+
```

Comment:
Can you move your code to after the comment so that all your codes are after the comments. This will help readability